features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given("the following user exists:") do |table|
+  table.hashes.each do |table|
+    User.create(table)
+  end
+end
+
+Given("I visit the page") do
+  visit '/'
+end
+
+When("I click on {string}") do |login|
+  click_on(login)
```

Comment:
please be consistent withpoetry mode or not, and change the iteration variable to something more generic for later use